docs/available_commands.md

```diff
@@ -20,6 +20,18 @@ Currently, the command adds or modifies the following pieces of information:
   * *externalReferences* of type *vcs*
 * Generates a *bom-ref* for components which don't have one, yet. The *bom-ref* will be a GUID.
 
```

Comment:
Please add the information that text is included (we are also missing that names are replaced with id, should include this via a different PR)